hq/test/logic/SnykDisplayTest.scala

```diff
@@ -0,0 +1,211 @@
+    organisationId.value() shouldBe "1111111111"
+  }
+
+  "fail to find organisationId (nice)" in {
+    val organisationId = SnykDisplay.getOrganisationId(SnykDisplayTest.mockBadResponseWithMessage, Organisation("guardian"))
+    organisationId.isFailedAttempt shouldBe true
+    organisationId.getFailedAttempt.failures.head.friendlyMessage shouldBe "Could not read Snyk response (some nice error)"
+  }
+
+  "fail to find organisationId (not nice)" in {
+    val organisationId = SnykDisplay.getOrganisationId(SnykDisplayTest.mockBadResponseWithoutMessage, Organisation("guardian"))
+    organisationId.isFailedAttempt shouldBe true
+    organisationId.getFailedAttempt.failures.head.friendlyMessage shouldBe """Could not read Snyk response ({"toughluck": "no use"})"""
+  }
+
+  "find project id list" in {
+    val projects = SnykDisplay.getProjectIdList(SnykDisplayTest.mockGoodProjectResponse)
+    !projects.value().exists(p => p.name == "project1") shouldBe false
+    !projects.value().exists(p => p.name == "project2") shouldBe false
+  }
+
+  "fail to find project id list (nice)" in {
+    val projects = SnykDisplay.getProjectIdList(SnykDisplayTest.mockBadResponseWithMessage)
+    projects.isFailedAttempt shouldBe true
+    projects.getFailedAttempt.failures.head.friendlyMessage shouldBe "Could not read Snyk response (some nice error)"
+  }
+
+  "fail to find project id list (not nice)" in {
+    val projects = SnykDisplay.getProjectIdList(SnykDisplayTest.mockBadResponseWithoutMessage)
+    projects.isFailedAttempt shouldBe true
+    projects.getFailedAttempt.failures.head.friendlyMessage shouldBe """Could not read Snyk response ({"toughluck": "no use"})"""
+  }
+
+  "find empty vulnerability list" in {
+    val projects = SnykDisplay.parseProjectVulnerabilities(List(SnykDisplayTest.mockGoodAndNotVulnerableResponse))
+    projects.value().head.ok shouldBe true
+  }
+
+  "find non-empty vulnerability list" in {
+    val projects = SnykDisplay.parseProjectVulnerabilities(List(SnykDisplayTest.mockGoodButVulnerableResponse))
+    projects.value().head.ok shouldBe false
+    projects.value().head.high shouldBe 1
+    projects.value().head.medium shouldBe 0
+    projects.value().head.low shouldBe 0
+  }
+
+  "fail to find vulnerability list (nice)" in {
+    val projects = SnykDisplay.parseProjectVulnerabilities(List(SnykDisplayTest.mockBadResponseWithMessage))
+    projects.isFailedAttempt() shouldBe true
+    projects.getFailedAttempt.failures.head.friendlyMessage shouldBe "Could not read Snyk response (some nice error)"
+  }
+
+  "fail to find vulnerability list (not nice)" in {
+    val projects = SnykDisplay.parseProjectVulnerabilities(List(SnykDisplayTest.mockBadResponseWithoutMessage))
+    projects.isFailedAttempt() shouldBe true
+    projects.getFailedAttempt.failures.head.friendlyMessage shouldBe """Could not read Snyk response ({"toughluck": "no use"})"""
+  }
+
+  "label projects" in {
```

Comment:
This test message isn't very useful! What does it do / what property are we checking?